.stickler.yml

```diff
@@ -0,0 +1,11 @@
+linters:
+  eslint:
+    config: './eslint.config'
+
+files:
+  ignore:
+      - 'dist/*'
+      - 'src/index.js'
+      - 'webpack.config.js'
+fixers:
+    enable: true
```

Comment:
Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/javascript).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))